index.css

```diff
@@ -85,7 +85,7 @@ section {
 	display: block;
 	float: left;
 	text-align: center;
-	max-width: calc(var(--max-content-width)/3);
+	max-width: 28em;
 	box-sizing: border-box;
```

Comment:
The sections were originally about 23 em wide. That seems enough to me. Why do you want to make them wider?